File: 1988-minimize-maximum-pair-sum-in-array/1988-minimize-maximum-pair-sum-in-array.java
class Solution {
    public int minPairSum(int[] nums) {
        Arrays.sort(nums);
        int t=nums.length;
        int max=0;
        for(int i=0;i<t/2;i++){
           int sum=nums[i]+nums[t-i- 1];
            if(max<sum){
                max=sum;
            }
        }
        return max;

        
    }
}

File: 1988-minimize-maximum-pair-sum-in-array/1988-minimize-maximum-pair-sum-in-array.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        nums.sort()
        t=len(nums)-1
        max=0
        for i in range(len(nums)//2):
            sum=nums[i]+nums[t-i]
            if max<sum:
                max=sum
        return max

